### PR TITLE
Feature - new icons / dropdown UX

### PR DESCRIPTION
**Pull request by mara1esh**

### CODE REVIEW

File: src/app/components/blocks/Sidebar.Links.tsx [hunk 1]
```diff
@@ -4,16 +4,22 @@ import { ReactComponent as OverviewIcon } from "app/icons/Overview.svg";
...
 import { ReactComponent as ContactsIcon } from "app/icons/Contacts.svg";
 import { ReactComponent as WalletsIcon } from "app/icons/Wallets.svg";
 import { ReactComponent as SettingsIcon } from "app/icons/Settings.svg";
 import { ReactComponent as SupportIcon } from "app/icons/Support.svg";
+import { ReactComponent as ActivityIcon } from "app/icons/ActivityIcon.svg";
 import * as SupportAlert from "app/components/elements/SupportAlert";
 import { useDialog } from "app/hooks/dialog";
+import { useSetAtom } from "jotai";
+import { activityModalAtom } from "app/atoms";
+import { useActivityBadge } from "app/hooks";
```
Comment (serg-plusplus) [inline src/app/components/blocks/Sidebar.Links.tsx]: Import ordering

File: src/app/components/blocks/Sidebar.Links.tsx [hunk 2]
```diff
@@ -4,16 +4,22 @@ import { ReactComponent as OverviewIcon } from "app/icons/Overview.svg";
...
 
 const useSidebarLinks = () => {
   const { alert } = useDialog();
+  const setActivityOpened = useSetAtom(activityModalAtom);
+  const showActivityBadge = useActivityBadge();
```
Comment (serg-plusplus) [inline src/app/components/blocks/Sidebar.Links.tsx]: "showActivityBadge" sounds like action/function. Better to call boolean vars like "activityBadgeDisplayed" | "activityBadgeShowed"

File: src/app/components/blocks/Sidebar.tsx [hunk 3]
```diff
@@ -65,43 +65,40 @@ const SidebarBlock: FC<SidebarBlockProps> = ({ links, className }) => {
...
+              <BadgeWrapper showBadge={badge}>
+                <Icon
+                  className={classNames(
+                    "w-7 h-7",
+                    "min-w-7",
+                    "styled-icon",
+                    isPageActive
+                      ? "styled-icon--active"
+                      : "group-hover:styled-icon--hover group-focus:styled-icon--hover",
+                  )}
```
Comment (serg-plusplus) [inline src/app/components/blocks/Sidebar.tsx]: The the code as used for link. Try to reuse shared code, for example:
```ts
const Component = isButton ? "button" : Link;

return (<Component {...props} />)
```

**Review by serg-plusplus — CHANGES_REQUESTED** [review]

File: src/app/components/blocks/Sidebar.tsx (hunk 3 above)
Comment (mara1esh) [reply src/app/components/blocks/Sidebar.tsx]: Well, I moved easy-to-reuse code into <LinkContent /> and provided it as button's child.
I don't think either Link and button has many common props to to smth like `<Component {...props} />`

**Pull request merged**